docs/sosyal-aglar.md

```diff
@@ -58,18 +56,13 @@ Bluesky'nin temel özellikleri:
 | --- | --- | --- | --- | --- |
 | <span style="color: green;">✓</span> | <span style="color: green;">✓</span> | MIT | <i class="fa-solid fa-globe"></i> <i class="fa-brands fa-android"></i> <i class="fa-brands fa-app-store-ios"></i> | 24 Milyon |
 
-#### Kaynaklar ve Dökümanlar
-
 - [Açık kaynak depo](https://github.com/bluesky-social/social-app)
 - [Bluesky Geliştirici Belgeleri](https://atproto.com/)
 - [Resmi Web Sitesi](https://bsky.app/)
 
-
 ### <span style="display: inline-block; vertical-align: middle;"><img src="docs/images/odyseelogo.png" alt="Odysee" style="width: 50px; height: auto;"> </span> <span style="display: inline-block; vertical-align: middle;"> Odysee
 
-**Odysee**, blockchain tabanlı bir merkeziyetsiz video paylaşma platformudur. **LBRY** (Library Client) sistemini kullanmaktadır, kullanıcılar videoları Odysee'ye yükler ve bu videolar dağıtık halde depolanır, yani bir merkezde tutulmaz. Odysee, MIT lisansını kullanır, yani Açık Kaynaklı bir platformdur. Videolarda reklam göstermez, Odysee kullanıcıları video izleyerek LBRY coin kazanır ve bu coinleri sevdiği üreticilere verebilir. Bir üretici aldığı Library Coin miktarı kadar platformda önerilir, böylece clickbait vb. durumlarının önüne geçilir. Odysee platformuna **YouTube** kanalları otomatik olarak bağlanılabilir.
-
-#### Kaynaklar
+[**Odysee**](https://odysee.com/), blockchain tabanlı bir merkeziyetsiz video paylaşma platformudur. **LBRY** (Library Client) sistemini kullanmaktadır, kullanıcılar videoları Odysee'ye yükler ve bu videolar dağıtık halde depolanır, yani bir merkezde tutulmaz. Odysee, MIT lisansını kullanır, yani Açık Kaynaklı bir platformdur. Videolarda reklam göstermez, Odysee kullanıcıları video izleyerek LBRY coin kazanır ve bu coinleri sevdiği üreticilere verebilir. Bir üretici aldığı Library Coin miktarı kadar platformda önerilir, böylece clickbait vb. durumlarının önüne geçilir. Odysee platformuna **YouTube** kanalları otomatik olarak bağlanılabilir.
```

Comment:
Diğer kısımlarda da [Oxford comma](https://www.kuttura.com/tr/oxford-comma-virgul) denilen virgül çok fazla kullanılmıştı fakat bu kısımda anlamı bozar hale gelmiş. Nokta yerine virgül koyulmuş gibi olmuş.

Reply:
"," bu sayfada kullanılan
"," bu benim klavyemde olan, bir farkı var mı? Çok anlamadım ne oduğunu tam olarak. Farklı bir virgük stili mi kullanılmış ilgili sayfada?

Reply:
Hayır hayır ikisi de aynı virgül. Türkçe yazım kurallarında olmayan ama ingilizcede olan bir cümle yapısı bu. Hani chatgpt'nin ürettiği bir içeriği okursunuz ve anlarsınız ya onun ürettiğini. Fakat nasıl anladınız bilemezsiniz. İşte bu oxford comma yüzünden.

Mesela (biraz değiştirilmiş bir metin olacak ama): 

> .... göstermez, ve .... kazanır 

şimdi burada tek başına `,` veya `ve` kullanmamız gerekirken ikisini bir anda kullandığımızı hayal edelim. Türkçede böyle bir kullanım normalde yok. Fakat ingilizceden virgül ve hemen ardından ve kullanımına yönelik bir imkan var. Malum chatgpt de internetteki tüm verilere hakim olduğu için günün sonunda bu tarz bir yazımı benimsemiş durumda. Bu durum eğer içeriği İngilizce ürettirirseniz direkt bu şekilde cereyan ederken Türkçede ise virgül kullanılmaması gereken yerlerde kullanma şeklinde kendini gösteriyor. _(Aslında anti-chatgpt programlarının hepsi bu özelliği kullanarak tespit yapıyor.)_

> Videolarda reklam göstermez, Odysee kullanıcıları video izleyerek LBRY coin kazanır ve bu coinleri sevdiği üreticilere verebilir.

Aynı örneğin bu sefer orijinal versiyonuna bakarsak burada tek başına `ve` olsaydı normal olacakken virgül hem okunurluluğu azaltmış hem de noktalama yanlışlığına neden olmuş oluyor.